src/Simulation/QCTraceSimulator/DepthCounter.cs

```diff
@@ -87,8 +87,8 @@ public void OnOperationEnd(object[] returnedQubitsTraceData)
             OperationCallRecord caller = operationCallStack.Peek();
             HashedString callerName = caller.OperationName;
 
-            caller.ReleasedQubitsAvailableTime = Max(opRec.ReleasedQubitsAvailableTime, caller.ReleasedQubitsAvailableTime );
-            caller.ReleasedQubitsAvailableTime = Max(opRec.ReleasedQubitsAvailableTime, caller.ReleasedQubitsAvailableTime );
```

Comment:
Is this a bug?

Reply:
Could you please provide a description for the PR? Something along the lines of "Removed duplicated code and unnecessary namespace includes. Fixed bug in calculation of depth for borrowed qubits."

Is it possible to add a test that fails prior to this fix and passes after?

Reply:
Yep. Good catch.